pkg/config/sync.go

```diff
@@ -276,6 +289,14 @@ func logCurrentConfig() {
 			config = append(config, fmt.Sprintf("workspace.ignoredUnrecoverableEvents=%s",
 				strings.Join(Workspace.IgnoredUnrecoverableEvents, ";")))
 		}
+		if Workspace.DefaultStorageSize != nil {
+			if Workspace.DefaultStorageSize.Common != nil {
+				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.common=%s", Workspace.DefaultStorageSize.Common.String()))
+			}
+			if Workspace.DefaultStorageSize.PerWorkspace != nil {
+				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.per-workspace=%s", Workspace.DefaultStorageSize.PerWorkspace.String()))
```

Comment:
```suggestion
				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.perWorkspace=%s", Workspace.DefaultStorageSize.PerWorkspace.String()))
```

Reply:
Done :) Will autosquash when ready for merge

Reply:
Should be good for merge assuming nothing else needs to be changed :sunglasses: